Dockerfile

```diff
@@ -6,20 +6,17 @@ LABEL Marcel Belmont "marcelbelmont@gmail.com"
 ENV appDir /var/www/app
 ENV NODE_SASS_PLATFORM alpine
 
-# Run updates and install deps
-RUN apk add --no-cache make gcc g++ python
-
 # Set the work directory
 RUN mkdir -p ${appDir}
 WORKDIR ${appDir}
 
+# Add application files
+ADD . ${appDir}
+
```

Comment:
Add files before installing `node_modules`, I considered adding just the `package.json`, but it's useful to have everything there just in case you ever wanted to build and publish the image.